File: WebApiUnitTesting/UnitTest1.cs
using AngularWebApi.Data;
using AngularWebApi.Data.Models;
using Microsoft.EntityFrameworkCore;
using Microsoft.Extensions.Configuration;
using NUnit.Framework;
using System.Collections;
using WebApi.Data;
using WebApi.Data.Models;

namespace WebApiUnitTesting
{
    public class Tests
    {
        private ArrayList repos;
        private WebApiDbContext context;

        [SetUp]
        public void Setup()
        {
            context = new WebApiDbContext(new DbContextOptionsBuilder<WebApiDbContext>()
                .UseSqlServer(
                    new ConfigurationBuilder()
                    .AddJsonFile("appsettings.json")
                        .Build()
                        .GetSection("ConnectionStrings")
                        .GetSection("TestConnection")
                        .Value)
                    .Options);
            repos = new ArrayList()
            {
                new ContentRepository<Image>(context),
                new ContentRepository<Audio>(context),
                new ContentRepository<Video>(context)
            };
        }

        [Test]
        public void Test1()
        {
            Assert.Pass();
        }
    }
}

File: WebApi.Tests/UnitTest1.fs
module WebApi.Tests

open NUnit.Framework
open WebApi.Data
open AngularWebApi.Data
open Microsoft.EntityFrameworkCore
open Microsoft.Extensions.Configuration
open WebApi.Data.Models
open AngularWebApi.Data.Models


let context = new WebApiDbContext(
                let builder = new DbContextOptionsBuilder<WebApiDbContext>()
                builder.UseSqlServer(
                    let cfg = new ConfigurationBuilder()
                    cfg
                        .AddJsonFile("appsettings.json")
                            .Build()
                        .GetSection("ConnectionStrings")
                        .GetSection("TestConnection").Value).Options)

type Repositories = {
    Image : ContentRepository<Image>
    Audio : ContentRepository<Audio>
    Video : ContentRepository<Video>
} 


[<SetUp>]
let Setup () = ()
   


[<Test>]
let Test1 () =
    Assert.Pass()